src/main.cc

```diff
@@ -671,6 +673,9 @@ main(int argc, char **argv)
 			/* Force interactive mode */
 			interactive = true;
 			break;
+		case 'd':
```

Comment:
To have some food for tomorrow discussions I've sketched commit which is introducing `-d` for debugger shell activation. Please see 4a3208cec9f2fd3de97e8619fd4d1eadaca45dea

Reply:
Deleted from the current PR